roles/network-multus/tasks/provision.yml

```diff
@@ -0,0 +1,77 @@
+---
+- name: oc Command
+  set_fact:
+    cluster_command: oc
+  when: cluster=="openshift"
+
+- name: kubectl Command
+  set_fact:
+    cluster_command: kubectl
+  when: cluster=="kubernetes"
+
+- name: Login As Super User
+  shell: "oc login -u {{ admin_user }} -p {{ admin_password }}"
+  when: cluster=="openshift"
+    and admin_user is defined
+    and admin_password is defined
+
+- name: Check if namespace "{{ namespace }}" exists
+  shell: "{{ cluster_command }} get ns | grep -w {{ namespace }} | awk \'{ print $1 }\'"
+  register: ns
+
+- name: Create {{ namespace }} namespace
+  shell: kubectl create namespace {{ namespace }}
+  when: ns.stdout != namespace
```

Comment:
ns.stdout includes newline, doesn't it? If it does, this would always pass as true